### DISCUSSION TITLE
Plataformas de estudio 💯

### DISCUSSION THREAD

**Discussion by doguedogue** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#295] [Ideas]
Existen muchas plataformas para estudiar. 
Para mi las mejores son las gratuitas 🚨:
- [Youtube](https://youtube.com)
- [Free code camp](https://www.freecodecamp.org/)
- [W3Schools](https://www.w3schools.com/)
- [Mozilla Developer](https://developer.mozilla.org/es/docs/Web/CSS/Tutorials)
- etc.

Sin embargo, siempre las que son de paga como que le echan mas ganitas, 
y también combinan una parte gratis y otra de paga para que le des una probada, ejemplo:
- [Codigo facilito](https://codigofacilito.com/)
- [Udemy](https://www.udemy.com/es/)
- [Platzi](https://platzi.com/r/leafar/) ** Plataforma Abierta Gratis del 18-20 de Marzo ⚠** 
  Yo personalmente estoy suscrito a esta y me ha servido en las semanas de estudio para complementar
- etc.

**Comment by JuanSalvador73** [comment]
Mil gracias por el aporte :D
